src/MSBuild/app.amd64.config

```diff
@@ -39,8 +39,8 @@
 
         <dependentAssembly>
```

Comment:
It's a bummer this all needs to be maintained manually, rather than relying on MSBuild's own automatic calculation.

Reply:
The automatic calculation burned us many times with the need to match VS stuff. Sadly I think manual is the best option at the moment given our constraints.